package.json

```diff
@@ -14,8 +14,8 @@
     "react-navigation": "1.0.0-beta.21",
     "react-redux": "^5.0.6",
     "redux": "^3.7.2",
+    "redux-persist": "^5.9.1",
     "redux-thunk": "^2.2.0",
-    "redux-persist": "^4.10.2",
```

Comment:
*NOTE* there's been an upgrade to `redux-persist` since I first installed it. The new version has great docs and is easy to use. Make sure you `$ npm install` after you pull down these changes to have the latest version of `redux-persist`